in_text/text_data_release.yml

```diff
@@ -36,208 +36,464 @@ file-format: Four zipped folders and associated files.
 
 entities:
   -
-    data-name: 01_fetch.zip
+    data-name: README.md
+    data-description: >-
+      A markdown file detailing how to run the included workflow.
+  -
+    data-name: environment.yaml
+    data-description: >-
+      An environment yaml for creating a conda environment with the necessary libraries.
+  -
+    data-name: Snakefile_fetch_munge
+    data-description: >-
+      A snakemake workflow used to fetch and process data for machine learning model.
+  -
+    data-name: Snakefile_model_analysis
+    data-description: >-
+      A snakemake workflow used to analyze model performance using information theory analysis, expected gradients, and other techniques.
+  -
+    data-name: Snakefile_run_ml_model
+    data-description: >-
+      A snakemake workflow used to run machine learning model.
+  -
+    data-name: 01_fetch
+    data-description: >-
+      A directory containing data and code to retrieve data for machine learning model.
+  -
+    data-name: 01_fetch/params_config_fetch_noaa_nerrs.yaml
+    data-description: >-
+      A configuration file defining read, write and save location of NOAA NERRS meteorological data.
+  -
+    data-name: 01_fetch/params_config_fetch_noaa_nos.yaml
+    data-description: >-
+      A configuration file defining read, write and save location NOAA NOS tidal data.
+  -
+    data-name: 01_fetch/params_config_fetch_usgs_nwis.yaml
     data-description: >-
-      A zipped file containing data and code to retrieve additional data for machine learning model.
-    attributes:
+      A configuration file defining read, write and save location of USGS NWIS discharge data.
+  -
+    data-name: 01_fetch/wildcards_fetch_config.yaml
+    data-description: >-
+      A configuration file containing parameters to download NOAA NOS, NOAA NERRS and USGS NWIS data.
+  -
+    data-name: 01_fetch/in
+    data-description: >-
+      A directory to hold unprocessed input data for machine learning model that had to be manually downloaded.
+  -
+    data-name: 01_fetch/in/953860.zip
+    data-description: >-
+      A zipped directory containing meteorological data and metadata from the NOAA NERRS delsjmet station.
+  -
+    data-name: 01_fetch/src
+    data-description: >-
+      A directory containing python scripts for fetching and processing model data.
+  -
+    data-name: 01_fetch/src/fetch_coawst_model.py
+    data-description: >-
+      XX
+  -
+    data-name: 01_fetch/src/fetch_noaa_nerrs.py
+    data-description: >-
+      XX
+  -
+    data-name: 01_fetch/src/fetch_noaa_nos.py
+    data-description: >-
+      XX
+  -
+    data-name: 01_fetch/src/fetch_usgs_nwis.py
+    data-description: >-
+      XX
+  -
+    data-name: 01_fetch/src/utils.py
+    data-description: >-
+      A python script with helper functions for retrieving and processing data.
+  -
+    data-name: 01_fetch/out
```

Comment:
Is this directory made when you run the pipeline? I don't see that directory in the repo.

Reply:
Yes - I mentioned that in the description: 'This directory is created by running this archive's pipeline (documented in the README.md).' Let me know if you think that needs more clarification though!

Reply:
Got it. I see it now. I think the description is clear.